orangecontrib/text/twitter.py

```diff
@@ -64,68 +65,116 @@ class TwitterAPI:
         call `reset` method before searching or provide `collecting=False`
         argument to search method.
     """
+
     attributes = []
     class_vars = []
 
-    tv = data.TimeVariable('Date')
-    authors = [(data.DiscreteVariable('Author'),
-               lambda doc: '@' + doc.author.screen_name), ]
+    tv = data.TimeVariable("Date")
+    authors = [
+        (
+            data.DiscreteVariable("Author"),
+            lambda doc: "@" + doc.author.screen_name,
+        ),
+    ]
     metas = [
-        (data.StringVariable('Content'), lambda doc: doc.full_text if not
-         doc.retweeted else doc.text),
+        (
+            data.StringVariable("Content"),
+            lambda doc: doc.full_text if not doc.retweeted else doc.text,
+        ),
         (tv, lambda doc: TwitterAPI.tv.parse(doc.created_at.isoformat())),
-        (data.DiscreteVariable('Language'), lambda doc: doc.lang),
-        (data.DiscreteVariable('Location'), lambda doc: getattr(doc.place, 'country_code', None)),
-        (data.ContinuousVariable('Number of Likes', number_of_decimals=0),
-         lambda doc: doc.favorite_count),
-        (data.ContinuousVariable('Number of Retweets', number_of_decimals=0),
-         lambda doc: doc.retweet_count),
-        (data.DiscreteVariable('In Reply To'),
-            lambda doc: '@' + doc.in_reply_to_screen_name if doc.in_reply_to_screen_name else ''),
-        (data.DiscreteVariable('Author Name'), lambda doc: doc.author.name),
-        (data.StringVariable('Author Description'), lambda doc: doc.author.description),
-        (data.ContinuousVariable('Author Statuses Count', number_of_decimals=0),
-         lambda doc: doc.author.statuses_count),
-        (data.ContinuousVariable('Author Favourites Count', number_of_decimals=0),
-         lambda doc: doc.author.favourites_count),
-        (data.ContinuousVariable('Author Friends Count', number_of_decimals=0),
-         lambda doc: doc.author.friends_count),
-        (data.ContinuousVariable('Author Followers Count', number_of_decimals=0),
-         lambda doc: doc.author.followers_count),
-        (data.ContinuousVariable('Author Listed Count', number_of_decimals=0),
-         lambda doc: doc.author.listed_count),
-        (data.DiscreteVariable('Author Verified'), lambda doc: str(doc.author.verified)),
-        (data.ContinuousVariable('Longitude'),
-            lambda doc: coordinates_geoJSON(doc.coordinates)[0]),
-        (data.ContinuousVariable('Latitude'),
-            lambda doc: coordinates_geoJSON(doc.coordinates)[1]),
+        (data.DiscreteVariable("Language"), lambda doc: doc.lang),
```

Comment:
Oh god that gets long. 😱

Reply:
I shortened this list for more than 10 lines now. :) If you think it is still too long I can make it as it was before (cca 40 lines shorter than now).